apigentools/utils.py

```diff
@@ -323,9 +324,9 @@ def write_full_spec(spec_dir, spec_version, spec_sections, fs_path):
         if not os.path.exists(fpath):
             raise errors.SpecSectionNotFoundError(spec_version, filename, fpath)
         with open(fpath) as infile:
-            loaded = yaml.load(infile, Loader=CSafeLoader)
-            for k, v in loaded.get("paths", {}).items():
-                full_spec["paths"].setdefault(k, {})
+            loaded = yaml.load(infile.read(), Loader=yamlordereddictloader.SafeLoader)
```

Comment:
The loader should accept file to avoid loading all content to memory.
```suggestion
            loaded = yaml.load(infile, Loader=yamlordereddictloader.SafeLoader)
```